### DISCUSSION TITLE
404 error at start fresh install

### DISCUSSION THREAD

**Discussion by johnycage** [EasyEngine/easyengine#1710] [Support Requests]
I have just installed ee to create new wordpress site with cache on fresh Ubuntu OS. No other packages, plain vanila Ubuntu. 
Followed 2 famous easy engine commands. 
After 1st command, had to start docker service. Then use 2nd command
I got the successfull command prompt messege with user database credentials. But when I go to browser and type domain name it throws **404  not found error**.

**Comment by dhsathiya** [comment]
I was not able to reproduce it.

Details required:
1. logs from `/opt/easyengine/logs/ee.log` - If possible zip and upload here.
2. Hosting provider - eg. AWS, GCP, DigitalOcean etc.
